src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -46,6 +46,22 @@ class ActivityFeed extends React.Component<Props, State> {
 
     feedContainer: null | HTMLElement;
 
+    componentDidMount() {
+        if (this.feedContainer) {
+            this.feedContainer.scrollTop = this.feedContainer.scrollHeight;
+        }
+    }
+
+    componentDidUpdate(prevProps) {
+        if (
+            this.feedContainer &&
+            prevProps.feedItems === undefined &&
+            this.props.feedItems
```

Comment:
It seems a little odd to check for `undefined` explicitly above, then check truthiness here. Is there a scenario where `feedItems` is null or zero?

Reply:
It looks like feedItems can potentially be null. I'll adjust this to check for falsiness of prevProps.feedItems instead